avocado/plugins/runner.py

```diff
@@ -364,7 +364,12 @@ def run_suite(self, job, test_suite):
                                                           execution_order):
                 test_parameters = test_factory[1]
                 name = test_parameters.get("name")
-                test_parameters["name"] = TestID(index, name,
+                if test_suite.name is None:
+                    suite_prefix = "{}-{}".format(test_suite.name, index)
```

Comment:
This caught my attention now, is it correct? It will create a prefix like "None-some_index".

Reply:
No is not, I'm without coffee. Sorry.

Reply:
Hehehe, same here, sorry, I should have told you in the previous version.